dev-client/src/components/map/SiteMap.tsx

```diff
@@ -26,20 +79,29 @@ const SiteMap = memo(({sites, center}: SiteMapProps): JSX.Element => {
     });
   }, [center]);
 
-  let markers = sites.concat(temporarySites).map(site => {
-    return (
-      <MarkerView
-        coordinate={site.coordinates}
-        key={site.id}
-        allowOverlap={true}>
-        <MaterialIconButton
-          name="location-on"
-          iconButtonProps={{size: 'sm'}}
-          iconProps={{color: 'secondary.main'}}
-        />
-      </MarkerView>
-    );
-  });
+  const sitesFeature = useMemo(
+    () =>
+      siteFeatureCollection(
+        Object.values(sites).filter(site => !site.archived),
+      ),
```

Comment:
remark: I think it would make sense to store sites and archived sites in different parts of our redux store, and only query archived sites when we need to display them. But let's worry about that when we get to displaying archived sites.